src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xencoder/HaivisionXEncoderCommunicator.java

```diff
@@ -1177,18 +1255,24 @@ private void setActionAudioControl(AudioConfig audioConfig) {
 	/**
 	 * Save audio apply change
 	 *
-	 * @param data the data is request param
+	 * @param audioConfig the audioConfig is instance in AuidoConfig
```

Comment:
param audioConfig the audioConfig is instance in AuidoConfig => param audioConfig the audioConfig is instance in AudioConfig